app/controllers/registrations_controller.rb

```diff
@@ -35,9 +35,9 @@ def account_update_params
       :current_password)
   end
 
-  # redirect new adopter users to adopter_profile#new
+  # redirect new adopter users to adoptable pets page
   def after_sign_up_path_for(resource)
-    resource.adopter_account ? new_profile_path : root_path
+    resource.adopter_account ? adoptable_pets_path : root_path
```

Comment:
This is a valid path. But I got some questions.
Are we supposed to create the `adopter_account` from the admin dashboard?
I am not sure where to create the `adopter_account` since `user has_one :adopter_account, dependent: :destroy` and there seems to be no after callback to create an `adopter_account` for the user.

Reply:
Is the goal of this to make it so only adopters go into the browser page versus the staff? Honestly... I would just point to `adoptable_pets_path` without any logic to try to handle staff & adopter differently. Internally we've talked about coming back to address the staff post login experience.

What ya think?

Reply:
I agree. I will change `resource.adopter_account ? adoptable_pets_path : root_path` to `adoptable_pets_path`

Reply:
Excellent! Thanks for hearing me out :)